client/state/selectors/is-requesting-media-item.js

```diff
@@ -1,9 +1,13 @@
 /**
  * External dependencies
  */
-
 import { get } from 'lodash';
 
+/**
+ * Internal dependencies
+ */
+import 'calypso/state/media/init';
```

Comment:
Seems like this selector is completely unused. But it should be fine to remove it in another PR.